sdk/nodejs/admissionregistration/v1/MutatingWebhookConfiguration.ts

```diff
@@ -94,6 +94,13 @@ import { getVersion } from "../../version";
               opts.version = getVersion();
           }
 
-          super(MutatingWebhookConfiguration.__pulumiType, name, props, opts);
+          const _opts = pulumi.mergeOptions(opts, {
+              aliases: [
+                  { parent: opts.parent, type: "kubernetes:admissionregistration.k8s.io/v1:MutatingWebhookConfiguration", name: name },
+                  { parent: opts.parent, type: "kubernetes:admissionregistration.k8s.io/v1beta1:MutatingWebhookConfiguration", name: name },
```

Comment:
Aliases shouldn't have to re-specify the `parent` and `name` if they aren't overriding these.  I assume we can leave them off across all of these as we are never trying to change those part?

Reply:
Will test that out.

Reply:
Confirmed. Will update.